scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -394,6 +416,64 @@ class FossId(
                 "FossId returned a null scancode for an existing scan"
             }
         }
+        if (waitForResult) checkScan(scanCode)
```

Comment:
Why does this check now happen *after* getting the raw result?

Reply:
Heu, no it still happen before: the call is in `checkAndCreateScan` and the latter is called before `getRawResults`.

Reply:
What I meant is: Is it intended that the original `checkScan` call from line 328 is removed?

PS: What does "Heu" mean? 😀

Reply:
Yes it is intended because the call moved to `checkAndCreateScan` and `checkAndCreateDeltaScan`.
Heu = Hrm in french :)